Packs/CVSS/CVSSCalculator/CVSSCalculator.py

```diff
@@ -0,0 +1,333 @@
+import math
+
+
+def round_up(n):
+    if n is None:
+        return None
+    int_input = int(n * 100000)
+    if (int_input % 10000) == 00:
+        return int_input / 100000.0
+    else:
+        return math.floor((int_input / 10000) + 1) / 10.0
+
+args = demisto.args()
```

Comment:
move all the variables to `main()` function. Only constant variable should be defined out of `main`
And add unit tests please.

Reply:
Done.